lib/src/kvs/ds.rs

```diff
@@ -216,6 +219,44 @@ impl Datastore {
 		self
 	}
 
+	/// Set global auth config
+	pub fn auth(mut self, auth: bool) -> Self {
+		self.auth_enabled = auth;
+		self
+	}
+
+	pub fn is_auth_enabled(&self) -> bool {
+		self.auth_enabled
+	}
+
+	// Setup the initial credentials
+	pub async fn setup_initial_creds(&self, creds: Root<'_>) -> Result<(), Error> {
+		let mut txn = self.transaction(false, false).await?;
+
+		match txn.all_kv_users().await {
+			Ok(val) if val.is_empty() => {
+				warn!(
+					target: LOG,
+					"No root users found, create the initial user '{}'.", creds.username
+				);
+
+				let sql = format!(
+					"DEFINE USER {user} ON KV PASSWORD '{pass}'",
```

Comment:
I'm not sure if this is something we should worry about at this point but is `{pass}` safe from  SQL injections here? Also can the password contain `'` or will it break the query?

Reply:
for more context, this won't be the recommended way for setting up a production cluster. This is more a helper for people not interesting in IAM and that just want to quickly setup a dev/test SurrealDB instance.

We will advocate for using `DEFINE USER` statements instead of the cli args

Reply:
Also, this is only executed the first time, before there's any user or data in the system, so the actual security risk of sql injection is not meaningful.

Reply:
I thought as much but thought I would ask to be sure.